src/Symfony/Component/Cache/Psr16Cache.php

```diff
@@ -19,6 +20,12 @@
 use Symfony\Component\Cache\Exception\InvalidArgumentException;
 use Symfony\Component\Cache\Traits\ProxyTrait;
 
+if (class_exists(InstalledVersions::class) && InstalledVersions::isInstalled('psr/simple-cache')) {
+    if (version_compare(InstalledVersions::getVersion('psr/simple-cache'), '3.0.0', '>=')) {
```

Comment:
There's `InstalledVersions::satisfies()` for this kind of query: https://getcomposer.org/doc/07-runtime.md#knowing-whether-package-x-is-installed-in-version-y

Reply:
That would require adding `"composer/semver": "^3.0"` as a dependency of `symfony/cache`.  Is that acceptable?

Reply:
I wasn't aware of that. No, we should not do that.